decidim-meetings/app/controllers/decidim/meetings/meetings_controller.rb

```diff
@@ -104,7 +104,7 @@ def withdraw
       private
 
       def meeting
-        @meeting ||= Meeting.not_hidden.where(component: current_component).find(params[:id])
+        @meeting ||= Meeting.not_hidden.where(component: current_component).find(params[:id]) if params[:id]
```

Comment:
Generally we use `find_by(id: params[:id])` if we want to allow it to be nil as well.